docs/source/user-guide/tasks/download-data.rst

```diff
@@ -0,0 +1,37 @@
+===============================
+Downloading data into a project
+===============================
+
+Often data sets are too large to keep locally, so you may want
+to download them on demand.
+
+[@electronwill] In the following procedure, we say to execute
+the command from within the project directory, but the
+command includes an environmental variable that specifies
+the path, so it seems like it would not be necessary to
+execute the command from within the project directory.
+If you aren't sure, please @mention an SME.
```

Comment:
After the file is downloaded, we create the environment variable and save the path of the file in that variable for future use. So, it is necessary to move into the project directory before downloading the file.